test/expressions/generateExpressionTests.py

```diff
@@ -237,12 +240,12 @@ def save_tests_in_files(N_files, tests):
 
 def handle_function_list(functions_input, signatures):
     """
-    Handles list of functions, splitting elements between functions and signatures.
+    Handles list of functions, splitting elements between functions and signatures.  
```

Comment:
Remove spaces if you end up changing anything in this pull, otherwise leave them.

Reply:
fixed